src/obr/core/core.py

```diff
@@ -19,6 +23,16 @@ def parse_variables(in_str):
     return in_str
 
 
+def path_to_signac(path: str | Path) -> str:
```

Comment:
Not to be a nitpick, but the shorthand `X | Y` can be used since python 3.10, [see](https://docs.python.org/3/library/typing.html#typing.Union).

If the python versions stated in `setup.py`  are correct, using Python 3.10 features should be no problem, as far as I am aware.

(I will still change this in the next commit, though.)

Reply:
Yeah I guess the setup.py is not correct. Currently, we use obr on a machine which has only python3.9 so until they install python3.10 or newer we need to stick with the Union. If they update it I totally prefer the shorthand!